src/Compatibility/ApiDiff/Microsoft.DotNet.ApiDiff.Tool/Microsoft.DotNet.ApiDiff.Tool.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetToolMinimum)</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <!-- <IsPackable>true</IsPackable>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>genapidiff</ToolCommandName> -->
```

Comment:
These should get enabled?

Reply:
Yes. I had to disable them because otherwise it throws an error when I try to test it directly.

Reply:
It's working now. Not sure what was happening but the fact that it's supposed to be a tool, prevented me from running the exe directly. No longer happening.